tests/test_operator.py

```diff
@@ -232,6 +232,26 @@ def test_dimension_size_override(self, nt=100):
         op(a=a, time=5)
         assert(np.allclose(a.data[0], 4.))
 
+    def test_override_composite_data(self):
+        original_coords = (1., 1.)
+        new_coords = (2., 2.)
+        p_dim = Dimension('p_src')
+        ndim = len(original_coords)
```

Comment:
Optional style nitpicks: All these variables could be simply used in-place in the `PointData` constructors below.